x-pack/test/alerting_api_integration/spaces_only/tests/actions/execute.ts

```diff
@@ -95,6 +96,7 @@ export default function ({ getService }: FtrProviderContext) {
         outcome: 'success',
         message: `action executed: test.index-record:${createdAction.id}: My action`,
         startMessage: `action started: test.index-record:${createdAction.id}: My action`,
+        source: ActionExecutionSourceType.HTTP_REQUEST,
```

Comment:
Doesn't seem like we have explicit tests for notification?  I imagine there must be a cases FT to test changing assignees that presumably would check to make sure the notification was attempted.  That we could add an event log check to.  If not, feels like we should open an issue to add one, unless it's super-simple to do in this PR.

Reply:
I looked up the PR for notifications and there's a comment in the description that there were no integration tests: https://github.com/elastic/kibana/pull/144391 

> I did not add integration tests due to the complexity of simulating an email server. I added unit test coverage. If integration test coverage is needed we can add the tests on another PR.

I can create a followup issue for it.

Reply:
Ya, please do, and reference that PR.  I think we have a fairly simple way of simulating an email server, maybe it's not appropriate for this behavior though.

Reply:
Created issue here: https://github.com/elastic/kibana/issues/152950